### ISSUE TITLE
Atom.Object.defineProperty.get is deprecated.

### ISSUE THREAD

**Issue by danward79** [jamischarles/atom-todo-show#37]
atom.workspaceView is no longer available.
In most cases you will not need the view. See the Workspace docs for
alternatives: https://atom.io/docs/api/latest/Workspace.
If you do need the view, please use `atom.views.getView(atom.workspace)`,
which returns an HTMLElement.
```
Atom.Object.defineProperty.get (/Applications/Atom.app/Contents/Resources/app/src/atom.js:54:11)
Object.activate (/Users/danward/.atom/packages/todo-show/lib/show-todo.coffee:33:9)
```

**Comment by jamischarles** [comment]
Is this still present with 0.8.0?

**Comment by danward79** [comment]
Ah, it looks like for some reason Atom was not automatically updating from 0.6.0. So I missed the fix. Manually Uninstalled and Re-Installed now to fix. Thanks

**Comment by diracdeltafunk** [comment]
I can confirm that this is present with 0.8.0. Here are my errors:

----

atom.workspaceView is no longer available.
In most cases you will not need the view. See the Workspace docs for
alternatives: https://atom.io/docs/api/latest/Workspace.
If you do need the view, please use `atom.views.getView(atom.workspace)`,
which returns an HTMLElement.
```
Atom.Object.defineProperty.get (/usr/share/atom/resources/app/src/atom.js:54:11)
ShowTodoView.openPath (/home/ben/.atom/packages/todo-show/lib/show-todo-view.coffee:260:9)
```

----

atom.workspaceView is no longer available.
In most cases you will not need the view. See the Workspace docs for
alternatives: https://atom.io/docs/api/latest/Workspace.
If you do need the view, please use `atom.views.getView(atom.workspace)`,
which returns an HTMLElement.
```
Atom.Object.defineProperty.get (/usr/share/atom/resources/app/src/atom.js:54:11)
ShowTodoView.moveCursorTo (/home/ben/.atom/packages/todo-show/lib/show-todo-view.coffee:269:25)
```

**Comment by MRodalgaard** [comment]
Closed by 0d966b6755f6cf01149502410828714119ff39a2.
